set.mm

```diff
@@ -98434,6 +98434,345 @@ a Cantor normal form (and injectivity, together with coherence
+       Fenton, 2-Feb-2011.) $)
+    trpredeq2d $p |- ( ph -> TrPred ( R , A , X ) = TrPred ( R , B , X ) ) $=
+      ( wceq ctrpred trpredeq2 syl ) ABCGBDEHCDEHGFBCDEIJ $.
+  $}
+
+  ${
+    trpredeq3d.1 $e |- ( ph -> X = Y ) $.
+    $( Equality deduction for transitive predecessors.  (Contributed by Scott
+       Fenton, 2-Feb-2011.) $)
+    trpredeq3d $p |- ( ph -> TrPred ( R , A , X ) = TrPred ( R , A , Y ) ) $=
+      ( wceq ctrpred trpredeq3 syl ) ADEGBCDHBCEHGFBCDEIJ $.
+  $}
+
+  ${
+    $d R a y i $.  $d A a y i $.  $d X a y i $.  $d Y a y i $.
+    $( A class is a transitive predecessor iff it is in some value of the
+       underlying function.  This theorem is not really meant to be used
+       directly: instead refer to ~ trpredpred and ~ trpredmintr .
+       (Contributed by Scott Fenton, 28-Apr-2012.) $)
+    eltrpred $p |- ( Y e. TrPred ( R , A , X ) <->
+                    E. i e. _om Y e. ( ( rec ( ( a e. _V |->
+     U_ y e. a Pred ( R , A , y ) ) , Pred ( R , A , X ) ) |` _om ) ` i ) ) $=
+      ( ctrpred wcel com cv cvv cpred ciun cmpt crdg cres cfv wrex dftrpred2
+      eleq2i eliun bitri ) FBCEHZIFDJDKGLAGKBCAKMNOBCEMPJQRZNZIFUEIDJSUDUFFABCD
+      EGTUADFJUEUBUC $.
+  $}
+
+  ${
+    $d A a e j $.  $d A a e y $.  $d B j $.  $d i j $.  $d j y $.
+    $d R a e j $.  $d R y $.  $d X a e j $.
+    $( Technical lemma for transitive predecessors properties.  All values of
+       the transitive predecessors' underlying function are subsets of the base
+       set.  (Contributed by Scott Fenton, 28-Apr-2012.) $)
+    trpredlem1 $p |- ( Pred ( R , A , X ) e. B ->
+                      ( ( rec ( ( a e. _V |-> U_ y e. a Pred ( R , A , y ) ) ,
+                                Pred ( R , A , X ) ) |` _om ) ` i ) C_ A ) $=
+      ( vj ve cv com wcel cpred cvv cfv wss c0 wceq eqsstrdi nfcv nn0suc predss
+      ciun cmpt crdg cres wi csuc wrex wo fr0g fveq2 sseq1d syl5ibr nfrdg nfres
+      wa nfmpt1 nffv predeq3 cbviunv mpteq2i rdgeq1 reseq1 mp2b iuneq1 frsucmpt
+      nfiun iunss a1i mprgbir wn frsucmptn adantl pm2.61dan adantr wb rexlimiva
+      0ss mpbird a1d jaoi syl nfvres pm2.61i ) EJZKLZBDFMZCLZWFGNAGJZBDAJZMZUCZ
+      UDZWHUEZKUFZOZBPZUGZWGWFQRZWFHJZUHZRZHKUIZUJWSHWFUAWTWSXDWIWRWTQWPOZBPWIX
+      EWHBWHCWNUKBDFUBSWTWQXEBWFQWPULUMUNXDWRWIXCWRHKXAKLZXCUQWRXBWPOZBPZXFXHXC
+      XFIXAWPOZBDIJZMZUCZNLZXHXFXMUQXGXLBGWHXAIWJXKUCZXLWPNGWHTZGXATZIGXIXKGXAW
+      PGWOKGWHWNGNWMURXOUOGKTUPXPUSGXKTVHZWNGNXNUDZRWOXRWHUEZRWPXSKUFRGNWMXNAIW
+      JWLXKBDWKXJUTVAVBWHWNXRVCWOXSKVDVEZIWJXIXKVFZVGXLBPXKBPZIXIIXIXKBVIYBXJXI
+      LBDXJUBVJVKSXFXMVLZUQXGQBYCXGQRXFGWHXAXNXLWPXOXPXQXTYAVMVNBVSZSVOVPXCWRXH
+      VQXFXCWQXGBWFXBWPULUMVNVTVRWAWBWCWGVLZWRWIYEWQQBWFKWOWDYDSWAWE $.
+  $}
+
+  ${
+    $d R a y z $.  $d A a y z $.  $d X a y z $.
+    $( Assuming it exists, the predecessor class is a subset of the transitive
+       predecessors.  (Contributed by Scott Fenton, 18-Feb-2011.) $)
```

Comment:
```suggestion
    $( Assuming it is a set, the predecessor class is a subset of the class of transitive
       predecessors.  (Contributed by Scott Fenton, 18-Feb-2011.) $)
```
I know we use "exists" as "is a set", but since in nearby theorems, we deal a lot with classes, I think "is a set" is clearer.